.github/workflows/graphene_tests.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     strategy:
       matrix:
         llvm-version: [17, 18]
+        python-version: [3.12, 3.13.0-rc.1]
```

Comment:
Actually shall we just trust that python is forwards compatible? This seems like it'll double CI compute for quite a small benefit.

The proper action support is cool tho.

Reply:
Sure, although it's not always forwards compatible. We don't do anything weird so it's likely fine.